aggregate_metadata.py

```diff
@@ -1,27 +1,39 @@
 import json
 import os
 
+# Set up paths file direcrtorty 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I'd recommend making the path an argument and wrapping this all in a function. Default invocation of the script can be to call the function on the top-level of the repo. The reason for adding an argument is that we can then create some simple tests in a `tests/` subdirectory to make sure the script works.